### DISCUSSION TITLE
Impossible Inpainted a girl or a woman

### DISCUSSION THREAD

**Discussion by Dindinweb** [lllyasviel/Fooocus#920] [General]
Hello and congratulations on this fantastic software. I have a problem, I can't create a man or a woman with inpaint tools In all versions ( v1 - v2.5 - v2.6 ) I can make animals, objects or other but not human. I tested inpaint with several models without success. . Has anyone succeeded? Thanks

**Comment by lllyasviel** [comment]
hi some tips to try:

1. check if the prompt is describing global image. so if your image has "one girl" and you want to add another "one girl", the prompt could be "two girl" (rather than mask some area and still use "one girl" as prompt)
2. check if the masked area is large enough to add another person.
3. use higher guidance scale, eg, bigger than 15
4. inpaint multiple times (inpaint, if the person is not generated, still darg the generated image to inpaint) sometimes works by magic
5. wait for Fooocus to provide better models.

**Reply by Dindinweb** [reply]
thank you, I'll follow your advice
